src/library/trackset/crate/cratefeature.cpp

```diff
@@ -228,15 +234,16 @@ bool CrateFeature::dragMoveAcceptChild(const QModelIndex& index, QUrl url) {
         return false;
     }
     Crate crate;
-    if (!m_pTrackCollection->crates().readCrateById(crateId, &crate) || crate.isLocked()) {
+    if (!m_pTrackCollection->crates().readCrateById(crateId, &crate) ||
+            crate.isLocked()) {
         return false;
     }
     return SoundSourceProxy::isUrlSupported(url) ||
             Parser::isPlaylistFilenameSupported(url.toLocalFile());
 }
 
-void CrateFeature::bindLibraryWidget(WLibrary* libraryWidget,
-        KeyboardEventFilter* keyboard) {
+void CrateFeature::bindLibraryWidget(
+        WLibrary* libraryWidget, KeyboardEventFilter* keyboard) {
```

Comment:
I have no Idea what happens here. I guess clang-format was confused because the line was broken unnecessarily before. You may unbreakable it manually. clan-format should not touch the line.